Add guidance for interruption panel contribution (#5262)
alphagov/govuk-design-system: src/components/panel/index.md

---
title: Panel

> description: The Panel component is a visible container used on confirmation or results pages

Reviewer: We wanna change this description in line with the updated lead paragraph

{% from "_example.njk" import example %}

The Panel component is a visible container used to highlight important content.
{{ example({ group: "components", item: "panel", example: "default", html: true, nunjucks: true, open: false, size: "m", loading: "eager" }) }}

## When to use this component

Use the Panel component to display important information within:

- [Confirmation pages](/patterns/confirmation-pages/), which tell the user they’ve successfully completed the transaction

> - [Interruption pages](/patterns/interruption-pages/), which pause the user journey to give important information

Reviewer 2: We use 'pause the user journey' and 'interrupt the user journey' in different places, I'm guessing this is intentional but raising just in case it makes sense for them to be consistent

Author: It was a bit of a stylistic choice on my part to avoid repeating "interrupt" all the time. I don't mind changing instances of 'interrupt the user journey' to keep things consistent.

## When not to use this component

> Never use the Panel component to highlight important information within body content.

Reviewer 2: The interruption panel sort of is 'body content' but in a blue box, could this guidance be confusing in Interruption panel context?

Author: I think the intention here was about just using it to visually styling text. I think we have a similar line for this around tables, let me dig that up...

Author: How about this? It's quite short and comes immediately after the bullet list under 'When to use', so it should read fairly straightforward.

```suggestion
Never use the Panel component to highlight any other information.
```
